### PR TITLE
Add site option

### PR DESCRIPTION
**Pull request by mikael10j**

### CODE REVIEW

File: gammaboard.py [hunk 1]
```diff
@@ -419,7 +419,7 @@ def plot_migration_matrices(exp, colorbar=True, **kwargs):
     return fig
 
 
-def create_resolution_fig():
+def create_resolution_fig(site='south'):
     """
     Create the figure holding the resolution plots for the dashboard
```
Comment (vuillaut) [inline gammaboard.py:424]: Would be good to edit the args in the docstring for the functions that have new parameters

File: gammaboard.py [hunk 2]
```diff
@@ -433,16 +433,20 @@ def create_resolution_fig():
     ax_imp_res = axes[1][0]
     ax_eff_area = axes[1][1]
 
-    ctaplot.plot_angular_res_cta_performance('north', ax=ax_ang_res, color='black')
-    ctaplot.plot_energy_resolution_cta_performances('north', ax=ax_ene_res, color='black')
-    ctaplot.plot_effective_area_performances('north', ax=ax_eff_area, color='black')
+    ctaplot.plot_angular_res_cta_performance(site, ax=ax_ang_res, color='black', label=site)
```
Comment (vuillaut) [inline gammaboard.py]: Why was the label not displayed in the figures in gammaboard?
Is it now?

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: gammaboard.py (hunk 1 above)
Comment (mikael10j) [reply gammaboard.py:424]: solved in c5a65dd

File: gammaboard.py (hunk 2 above)
Comment (mikael10j) [reply gammaboard.py]: The labels were indeed not displayed. This is now done thanks to the call to axes.legend().
The site argument has been removed in c5a65dd to not override the default one.

**Review by vuillaut — APPROVED** [review]

**Pull request merged**